dhis-2/dhis-services/dhis-service-administration/src/main/java/org/hisp/dhis/maintenance/jdbc/JdbcMaintenanceStore.java

```diff
@@ -117,8 +117,8 @@ public int deleteSoftDeletedEvents() {
           "delete from programmessage_phonenumbers where programmessagephonenumberid in "
               + pmSelect,
           // delete related PSIs comments
```

Comment:
`PSIs comments` to be `event notes`

Reply:
@lucaCambi77 the conversation was marked as resolved but the comment @enricocolasante mentioned is still here. maybe you forgot to push your changes 😅

Reply:
There are other comments using `PSI` in here, can you please replace them with `event`